pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -248,6 +254,12 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha2.NodeObs
 								},
 							},
 						},
+						{
+							Name: dataVolumeName,
+							VolumeSource: corev1.VolumeSource{
+								EmptyDir: &corev1.EmptyDirVolumeSource{},
+							},
+						},
 					},
 					NodeSelector: nodeObs.Spec.NodeSelector,
```

Comment:
Can you add termination grace period to the daemonset pod spec? (related to https://github.com/openshift/node-observability-agent/pull/38)

Reply:
Yes, sure will do: once the agent PR is merged, I'll bump the new version of the agent with the needed grace period.

Reply:
So since #110 is opened, will you add the change in that or this PR?

Reply:
It's already [there](https://github.com/openshift/node-observability-operator/blob/2e77565788be53849501e5143a87592d44f02c12/pkg/operator/controller/nodeobservability/daemonset.go#L141-L143).